checkov/json_doc/runner.py

```diff
@@ -85,3 +85,9 @@ def import_registry(self):
 
     def require_external_checks(self):
         return True
+
+    @staticmethod
+    def _change_files_path_to_relative(report: Report):
+        for record in report.get_all_records():
+            record.file_path = record.file_path.replace(os.getcwd(), '')
+            record.resource = record.resource.replace(os.getcwd(), '')
```

Comment:
Is it OS agnostic?

Reply:
@Saarett  what do u mean?

Reply:
Would it yield the same result for different operating systems? Windows, Linux....

Reply:
And what if the user would run it locally? Or is it not possible?

Reply:
it is possible. when the user run checkov cli. but in both cases `os.getcwd()` will be the same